### PR TITLE
Spelling: Support documentation reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs/admin/support.rst [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
 offerings at <https://weblate.org/support/>.
 
 .. _activate-support:
 
-Activating support
+Integrating support
 ------------------
 
-Since Weblate 3.8 the purchased support packages can be activated in a Weblate
-management interface. The activation will enable peridic registration on
```
Comment (comradekingu) [inline docs/admin/support.rst]: I was thinking "Doing so registers your instance on official Weblate servers.", but I don't know what it means…?

File: docs/admin/support.rst [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+
+Weblate is copylefted libre software with community support.
+Subscribers receive priority support at no extra charge. Prepaid help packages are
+available for everyone. You can find more info about current support
 offerings at <https://weblate.org/support/>.
 
 .. _activate-support:
 
-Activating support
+Integrating support
```
Comment (comradekingu) [inline docs/admin/support.rst]: Possibly conflicting

File: docs/admin/support.rst [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
-servers.
 
-.. image:: /images/support.png
+.. versionadded:: 3.8
 
-Data submitted to the server
-----------------------------
+Purchased support packages can optionally be integrated into your Weblate
+subscription management interface, from where you will find a link to it.
+Basic telemetry is also reported back to Weblate about your installation.
```
Comment (comradekingu) [inline docs/admin/support.rst]: Something other than "telemetry" would be great.

Comment (comradekingu) [inline docs/admin/support.rst]:
```suggestion
Basic instance details about your installation are also reported back to Weblate.
```

File: docs/admin/support.rst [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
-installation from subscription management and sends basic telemetry to Weblate
-servers.
 
-.. image:: /images/support.png
+.. versionadded:: 3.8
 
-Data submitted to the server
-----------------------------
+Purchased support packages can optionally be integrated into your Weblate
+subscription management interface, from where you will find a link to it.
```
Comment (comradekingu) [inline docs/admin/support.rst]: Is this https://weblate.org/user/ and if so, would a link be in order?

Comment (nijel) [reply docs/admin/support.rst]: Yes, that's it. Link would probably make sense here.

File: docs/admin/support.rst (hunk 1 above)
Comment (nijel) [reply docs/admin/support.rst]: It periodically sends basic details to the server to check support status. The server can also respond with configuration for some additional services. Currently, the only additional service is backup, see https://docs.weblate.org/en/latest/admin/backup.html#cloudbackup

File: docs/admin/support.rst (hunk 4 above)
Comment (comradekingu) [inline docs/admin/support.rst]:
```suggestion
 `subscription management <https://weblate.org/user/>`_ interface, from where you will find a link to it.
```

File: docs/admin/support.rst [hunk 5]
```diff
@@ -1,37 +1,43 @@
...
+Weblate is copylefted libre software with community support.
+Subscribers receive priority support at no extra charge. Prepaid help packages are
+available for everyone. You can find more info about current support
 offerings at <https://weblate.org/support/>.
 
 .. _activate-support:
 
-Activating support
+Integrating support
 ------------------
```
Comment (nijel) [inline docs/admin/support.rst]:
```suggestion
Integrating support
-------------------
```

File: docs/admin/support.rst [hunk 6]
```diff
@@ -1,37 +1,43 @@
...
+* URL where your Weblate isntance is configured
+* Your site title
+* The Weblate version you are running
+* Tallies of some objects in your Weblate database (projects, components, languages, source strings and users)
+* The public SSH key of your instance
 
 No other data is submitted.
 
+Integration services
+------------------------
```
Comment (nijel) [inline docs/admin/support.rst]:
```suggestion
Integration services
--------------------
```

File: docs/admin/support.rst [hunk 7]
```diff
@@ -1,37 +1,43 @@
...
-management interface. The activation will enable peridic registration on
-Weblate servers. This allows you to directly navigate to your Weblate
-installation from subscription management and sends basic telemetry to Weblate
-servers.
 
-.. image:: /images/support.png
+.. versionadded:: 3.8
+
+Purchased support packages can optionally be integrated into your Weblate
+ `subscription management <https://weblate.org/user/>`_ interface, from where you will find a link to it.
```
Comment (nijel) [inline docs/admin/support.rst]:
```suggestion
`subscription management <https://weblate.org/user/>`_ interface, from where you will find a link to it.
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**